src/encoded/static/components/static-pages/placeholders/MdSortableTable.js

```diff
@@ -0,0 +1,650 @@
+                descSortFunction: CustomSorter.desc,
+                ascSortFunction: CustomSorter.asc,
+                //render: (id) => { return (<a href={'user/' + id}>{id}</a>); }
+            };
+        });
+
+        return (
+            <SortableTable {...{ data, columns, defaultColWidths }} />
+        );
+    }
+}
+
+const CustomSorter = {
+    desc: (data, key) => {
+        const clone = [...data];
+        const result = clone.sort(function (_a, _b) {
+            const r = Utils.compareData(_a[key], _b[key]);
+            return r !== 1 ? 1 : -1;
+        });
+        return result;
+    },
+
+    asc: (data, key) => {
+        const clone = [...data];
+        const result = clone.sort(function (_a, _b) {
+            const r = Utils.compareData(_a[key], _b[key]);
+            return r !== -1 ? 1 : -1;
+        });
+        return result;
+    }
+};
+
+const Utils = {
+    /**
+     * forked and modified from https://github.com/jrf0110/convert-text-table/
+     * @param {*} input - markdown table input string
+     */
+    convertMarkdownTableToObject: (input) => {
+        let i = 1, columnBorder;
+        const out = [];
+
+        //workaround to run https://github.com/jrf0110/convert-text-table/ solution for standard markdown tables, since
+        //standard tables not use "+" for column headers. first convert any "|---|" to "+---+"
+        input = input.replace(/\|-{3}/g, "+---").replace(/-{3}\|/g, "---+");
+
+        columnBorder = input.substring(input.indexOf('+') + 1);
+        columnBorder = '+' + columnBorder.substring(0, columnBorder.indexOf('+') + 1);
+
+        input = input.split('|').map(function (v) { return v.trim(); });
+
+        while (input[i++].indexOf(columnBorder) == -1 && input.length > i);
+
+        const headers = input.slice(1, i - 1);
+
+        input = input.slice(i, input.length - 1);
+
+        let obj = {};
+        input.forEach(function (val, i) {
+            if (i % (headers.length + 1) === (headers.length)) {
+                out.push(obj);
+                return obj = {};
+            }
+
+            obj[headers[i % (headers.length + 1)]] = Number.isNaN(parseFloat(val)) ? val : parseFloat(val);
+        });
+        //push the last item
+        if (!_.isEmpty(obj)) {
+            out.push(obj);
+        }
+
+        return out;
+    },
+    /**
+     * https://github.com/stiang/remove-markdown/blob/master/index.js
+     * @param {*} md - markdown input
+     * @param {*} options - remove options
+     */
+    removeMarkdown: (md, options) => {
+        options = options || {};
+        options.listUnicodeChar = options.hasOwnProperty('listUnicodeChar') ? options.listUnicodeChar : false;
+        options.stripListLeaders = options.hasOwnProperty('stripListLeaders') ? options.stripListLeaders : true;
+        options.gfm = options.hasOwnProperty('gfm') ? options.gfm : true;
+        options.useImgAltText = options.hasOwnProperty('useImgAltText') ? options.useImgAltText : true;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not access Object.prototype method 'hasOwnProperty' from target object.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4682591)